src/App.js

```diff
@@ -9,20 +9,20 @@ class App extends React.Component {
       super()
       this.state = {
         movies: movieData.movies,
-        selectedMovie: ""
+        // selectedMovie: ""
       }
     }
 
-    returnHome = () => {
-      this.setState({selectedMovie: ""})
-    }
+    // returnHome = () => {
+    //   this.setState({selectedMovie: ""})
+    // }
 
-    showMovieDetails = (id) => {
-      const foundMovie = this.state.movies.find(movie => {
-        return movie.id === id
-      })
-      this.setState({selectedMovie: foundMovie})
-    } 
+    // showMovieDetails = (id) => {
+    //   const foundMovie = this.state.movies.find(movie => {
+    //     return movie.id === id
+    //   })
+    //   this.setState({selectedMovie: foundMovie})
+    // } 
```

Comment:
Good idea to just comment this out in case you will want to use it at a later time.

Reply:
Just don't forget to remove those comments before you submit! Usually, you don't want to push commented code like this to main - you'd keep it on a branch instead.